src/util/gdal_timesnap.cpp

```diff
@@ -110,13 +112,19 @@ GDALTimesnap::GDALDataLoadingInfo GDALTimesnap::getDataLoadingInfo(Json::Value d
 {
     // get parameters
     Json::Value channelJson = datasetJson["channels"][channel];
-
+    
+    // get parameters
 	std::string time_format = channelJson.get("time_format", datasetJson.get("time_format", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: time_format: ", time_format));
 	std::string time_start 	= channelJson.get("time_start", datasetJson.get("time_start", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: time_start: ", time_start));
 	std::string time_end 	= channelJson.get("time_end", datasetJson.get("time_end", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: time_end: ", time_end));
 
     std::string path 	 = channelJson.get("path", datasetJson.get("path", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: path: ", path));
     std::string fileName = channelJson.get("file_name", datasetJson.get("file_name", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: fileName: ", fileName));
```

Comment:
I guess you should reduce the log messages to a minimum. This spams the log.
For all introduced log messages: Are they really necessary or was it for debuggin a specific problem?

Reply:
well they are debug logs and help to figure out why a file is not loaded. They only appear if you switch logging to debug. then you want to see them...